roles/ipaserver/library/ipaserver_test.py

```diff
@@ -1055,6 +1055,10 @@ def main():
 
     domain_name = domain_name.lower()
 
+    if host_name.lower() == domain_name.lower():
```

Comment:
`domain_name` is already lower case, please have a look at line 1056.

Reply:
´host_name` is also lower case, please have a look at line 1044.

Reply:
fixed